Governance/STRATEGIC-PLANS.md

```diff
@@ -0,0 +1,33 @@
+## The Annual Strategic Plans
+
+-   [2019](./Strategic-Plan/2019.md)
+-   [2018](./Strategic-Plan/2018.md)
+-   [2017](./Strategic-Plan/2017.md)
```

Comment:
Please file an issue to add the 2020 and 2021 plans, and I'll dig them out!

Reply:
It was there in the `Strategic Plan` subdirectory. I have fixed them up.